spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter_TimestampSuite.scala

```diff
@@ -294,15 +293,16 @@ class StandardizationInterpreter_TimestampSuite extends AnyFunSuite with TZNorma
 
     val src = seq.toDF(fieldName)
 
+    spark.sql("set spark.sql.legacy.timeParserPolicy=LEGACY")
```

Comment:
Doesn't this change the behavior only for the test and the actual code would still do it wrong?

Reply:
At the moment, yes. I am not sure what should be the behaviour from our point of view